src/cljs/main/broadfcui/components/split_pane.cljs

```diff
@@ -27,6 +27,7 @@
          [:div {:style {:flex "0 0 10px"
                         :display "flex" :flexDirection (if left "column" "row") :justifyContent "center"
                         :backgroundColor (:background-light style/colors)
+                        :margin (str "0 " slider-padding)
```

Comment:
The slider is designed to work both vertically and horizontally, so you should check which sides to apply the padding:

```clojure
(if top (str slider-padding " 0") (str "0 " slider-padding))
```